### ISSUE TITLE
[wg/png] PNG Working Group Charter

### ISSUE THREAD

**Issue by svgeesus** [w3c/strategy#514]
New charter proposal, reviewers please take note.

# Charter Review

[Charter](https://w3c.github.io/charter-drafts/2025/png-wg.html)

- [diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/charter-drafts/2025/png-wg.html)

Expected end of charter refinement phase: [15 October 2025](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JulSep/0044.html)

If applicable:

- [diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FGraphics%2FPNG%2Fpng-2023.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fpng-wg.html)
- [chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/png)

 - [x] Existing WG recharter 

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/orgs/w3c/projects/97/views/2).**

## Known or potential areas of concern

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...)

This issue

## Anything else we should think about as we review? 

Note: proposed chairs should be copied @programmax

**Comment by svgeesus** [comment]
Reviewers, I [explained the changes we are considering for 4th Edition in more detail](https://github.com/w3ctag/design-reviews/issues/1143#issuecomment-3233007799) for the TAG

**Comment by svgeesus** [comment]
Charter refinement [Disposition of Comments](https://w3c.github.io/charter-drafts/2025/png-wg.bsi.html) started
